example/src/constants/screenStyles.js

```diff
@@ -1,20 +1,27 @@
-import { StyleSheet, Platform } from 'react-native'
-import { ifIphoneX } from '../constants/utils'
-import colors from './colors'
-import constants from './constants'
+import {StyleSheet, Platform} from 'react-native';
```

Comment:
I don't know if it's intentional, but I checked the rest of the code and there are no semicolons there. It looks like a whole file was parsed by wrong Prettier settings.

Reply:
done